docs/serving/batch-serving.md

```diff
@@ -0,0 +1,611 @@
+      <td>-2.835548</td>
+      <td>1.338381</td>
+      <td>-1.385303</td>
+      <td>-2.276456</td>
+      <td>0</td>
+    </tr>
+    <tr>
+      <th>2</th>
+      <td>-1.519005</td>
+      <td>-1.807549</td>
+      <td>0.697304</td>
+      <td>-1.118860</td>
+      <td>1.104900</td>
+      <td>0</td>
+    </tr>
+    <tr>
+      <th>3</th>
+      <td>-0.632087</td>
+      <td>-0.345659</td>
+      <td>0.244329</td>
+      <td>-0.046066</td>
+      <td>0.447280</td>
+      <td>0</td>
+    </tr>
+    <tr>
+      <th>4</th>
+      <td>-1.405883</td>
+      <td>-1.746045</td>
+      <td>0.653617</td>
+      <td>-1.110985</td>
+      <td>-1.675466</td>
+      <td>0</td>
+    </tr>
+    <tr>
+      <th>...</th>
+      <td>...</td>
+      <td>...</td>
+      <td>...</td>
+      <td>...</td>
+      <td>...</td>
+      <td>...</td>
+    </tr>
+    <tr>
+      <th>9995</th>
+      <td>-2.641530</td>
+      <td>-2.597802</td>
+      <td>1.151097</td>
+      <td>-1.382713</td>
+      <td>-0.253170</td>
+      <td>0</td>
+    </tr>
+    <tr>
+      <th>9996</th>
+      <td>-0.116548</td>
+      <td>-0.903265</td>
+      <td>0.139706</td>
+      <td>-0.874907</td>
+      <td>1.011517</td>
+      <td>0</td>
+    </tr>
+    <tr>
+      <th>9997</th>
+      <td>4.699669</td>
+      <td>2.120651</td>
+      <td>-1.765963</td>
+      <td>-0.121263</td>
+      <td>-0.674961</td>
+      <td>1</td>
+    </tr>
+    <tr>
+      <th>9998</th>
+      <td>1.151971</td>
+      <td>0.991788</td>
+      <td>-0.486083</td>
+      <td>0.457369</td>
+      <td>-1.734896</td>
+      <td>1</td>
+    </tr>
+    <tr>
+      <th>9999</th>
+      <td>-0.665400</td>
+      <td>-1.157263</td>
+      <td>0.346660</td>
+      <td>-0.867286</td>
+      <td>-1.903681</td>
+      <td>0</td>
+    </tr>
+  </tbody>
+</table>
+<p>10000 rows × 6 columns</p>
+</div>
+
+
+Next, we drop the label column 
+
+
+```python
+sample = mlrun.get_dataitem(DATASET_URL).as_df().head().drop("labels", axis=1)
+```
+
+
+```python
+sample = sample.values.tolist()
+sample
+```
+
+
+
+
+    [[-0.2651151839702463,
+      -1.9322600633270253,
+      0.3039917127899224,
+      -1.863833475610883,
+      -1.045634803356314],
+     [-3.1354794002707846,
+      -2.835547936064964,
+      1.3383808404873596,
+      -1.385302821687631,
+      -2.2764563589804325],
+     [-1.519004884164945,
+      -1.8075487992775987,
+      0.6973035703821813,
+      -1.1188598603515594,
+      1.1048998961504315],
+     [-0.6320867871545256,
+      -0.3456588580241298,
+      0.2443294437742147,
+      -0.0460657045357599,
+      0.4472804635971865],
+     [-1.405882596126037,
+      -1.7460450817666373,
+      0.6536171315150338,
+      -1.1109846976791669,
+      -1.6754663728281285]]
+
+
+
+## Run prediction
+
+you can either use a trained model that is already registered in MLRun or just pass the model URI
+
+
+```python
+model_path = 'store://artifacts/auto-trainer-admin/MyModel:cf9538d81a2a4ce9a2035ad3d8b90456'
```

Comment:
Should reference a model from the samples and not something that is not runnable if I take the code as-is